scripts/transformDeps.sh

```diff
@@ -1,4 +1,9 @@
-git checkout -b examples
+# this is ment to be run localy 
+git fetch
+git pull origin main
+git pull origin examples
```

Comment:
Wouldn't we get a lot of conflicts here since examples will have a lot of diffs in the `package.json` files?

Reply:
I have not gotten any so far. But there could be